pkgs/development/libraries/liberfa/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, autoreconfHook }:
+
+stdenv.mkDerivation rec {
+  pname = "erfa";
+  version = "1.7.0";
+
+  buildInputs = [ autoreconfHook ];
+
+  src = fetchFromGitHub {
+    owner = "liberfa";
+    repo = "erfa";
+    rev = "v${version}";
+    sha256 = "1z4k2phrw6wwi0kax6ac80jk9c036gi7pmhmg6gaf3lk81k6xz2r";
+  };
+
+  configureFlags = [ "--enable-shared" ];
+
+  meta = with stdenv.lib; {
+    description = "Essential Routines for Fundamental Astronomy";
+    homepage = "https://github.com/liberfa/erfa";
+    license = "licenses.erfa";
```

Comment:
- this should not be a string, but instead (if `erfa` is defined in `licenses`): `license = licenses.erfa`
- but, since the license is specific to this one package, it would be better to leave it out of `licenses` and just define it here (`license = { ... what you currently put in licenses ...}`).

Reply:
- yes of course, I removed with the quotes.
- so this would be possible pack the license information into the `default.nix`? It is true that this is a special license that will most probably not be used anywhere else. But I suppose this is valid for some others already defined there as well.

Reply:
Yes, that's possible. There is no difference between importing it from `licenses` or just specifying it directly.

> But I suppose this is valid for some others already defined there as well.

That may be true. nixpkgs is huge and there are a lot of good, bad and ugly examples for pretty much everything ;) I try to hold new additions to a higher standard.

Its better to include it in the file, since its very local to the package and might need to change with it / be removed with it. It also makes it very clear that this is a custom license.

Reply:
(At least that is my opinion in this case, other people who have added their one-off licenses to `licenses.nix` may have had valid reasons.)

Reply:
@timokau I fully agree. It helps a lot to increase the number of good examples.

Reply:
Great :) Ping me once you've made the changes, I don't trust github to notify me on new commits (sometimes it does, sometimes it doesn't).

Reply:
@timokau hopefully now everything is done :smile: